torch/distributed/rpc/api.py

```diff
@@ -358,19 +359,49 @@ def _validate_rpc_args(backend, store, name, rank, world_size, rpc_backend_optio
 GenericWithOneTypeVar = Generic[T]
 
 
-try:
-    class RRef(PyRRef, GenericWithOneTypeVar):
-        # Combine the implementation class and the type class.
-        pass
-except TypeError as exc:
```

Comment:
Why don't we need the try-except any more?

Reply:
Because I am removing their inheritance relation.

Reply:
Have to not use inheritance, because when I call `super(RRef, self).method_name(*args, **kwargs)`, pybind11 expects an exact type match of `PyRRef`, while `self` is of `RRef` at that moment.